src/renderer/components/+config-limit-ranges/limit-ranges.route.ts

```diff
@@ -0,0 +1,12 @@
+import type { RouteProps } from "react-router";
+import { buildURL } from "../../../common/utils/buildUrl";
+
```

Comment:
Perhaps a redundant empty line?

Reply:
thank you, @nevalla 
My bad, I've forgotten to format the code